boards/nrf9160dk/include/board.h

```diff
@@ -34,8 +34,10 @@ extern "C" {
  */
 #define CLOCK_HFCLK         (32U)           /**< set to  0: internal RC oscillator
                                              *        32: 32MHz crystal */
-#define CLOCK_LFCLK         (3)             /**< set to  0: internal RC oscillator
-                                             *         3: High Accuracy oscillator */
+/* LFCLK Source clock selection:*/
+/* - CLOCK_LFCLKSRC_SRC_LFRC: 32.768 kHz RC oscillator
+ * - CLOCK_LFCLKSRC_SRC_LFXO: 32.768 kHz crystal oscillator */
+#define CLOCK_LFCLK         (CLOCK_LFCLKSRC_SRC_LFXO) /**< LFCLK Source */
```

Comment:
This changed from 3 -> 2 but since it's part of the nRF53 family I assume you tested this and 3 was wrong before and just worked by chance